### PR TITLE
04-technical: formating grammar

### PR DESCRIPTION
**Pull request by danielsaggau**
formating, grammar, hyperlinks, and introduction modifications

### CODE REVIEW

File: bookdown/04-technical/04-technical.Rmd [hunk 1]
```diff
@@ -2,12 +2,39 @@
 
 This chapter gives an overview about technical details of the `r mlr_pkg("mlr3")` framework.
 
+**Parallelization**
+
 At first, some details about [Parallelization](#parallelization) and the usage of the `r cran_pkg("future")` are given.
-Parallelization refers to running multiple jobs simultaneously and is used to minimize necessary computer power. 
+Parallelization refers to the process of running multiple jobs simultaneously.
+Parallelization is used to minimize the necessary computer power.
```
Comment (pat-s) [inline bookdown/04-technical/04-technical.Rmd]: Two sentences starting with the same word. Maybe we can change this?

File: bookdown/04-technical/04-technical.Rmd [hunk 2]
```diff
@@ -2,12 +2,39 @@
...
+Irrespective, parallelization does not always minimize computer power.
```
Comment (pat-s) [inline bookdown/04-technical/04-technical.Rmd]: Is it called "computer power"? same one line above

File: bookdown/04-technical/04-technical.Rmd [hunk 3]
```diff
@@ -2,12 +2,39 @@
...
+Algorithms consist of both sequential and paralizable parts. 
+Therefore, parallelization does not always alter performance in a positive substantial manner.
+In some cases, parallelization increases the duration of the entire undertaking.
+Summed up, this sub-chapter illustrates how and when to use parallelization in mlr3.
+
```
Comment (pat-s) [inline bookdown/04-technical/04-technical.Rmd]: Are there too many pointers in the paragraph why parallelization might be "bad"? It reads a bit as that parallelization might be more worse than good. Maybe we can use only one side sentence for this statement or put it in brackets?

File: bookdown/04-technical/04-technical.Rmd [hunk 4]
```diff
@@ -2,12 +2,39 @@
...
+**Database Backends**
+
+The Section [Database Backends](#backends) describes how to work with database backends that `r mlr_pkg("mlr3")` supports.
+Database Backends can be helpful for large data processing which does not fit in memory or is stored natively in a database (e.g. sqlite).
```
Comment (pat-s) [inline bookdown/04-technical/04-technical.Rmd]:
```suggestion
Database backends can be helpful for large data processing which does not fit in memory or is stored natively in a database (e.g. sqlite).
```

File: bookdown/04-technical/04-technical.Rmd [hunk 5]
```diff
@@ -2,12 +2,39 @@
...
+Parallelization is used to minimize the necessary computer power.
+Irrespective, parallelization does not always minimize computer power.
+Algorithms consist of both sequential and paralizable parts. 
+Therefore, parallelization does not always alter performance in a positive substantial manner.
+In some cases, parallelization increases the duration of the entire undertaking.
+Summed up, this sub-chapter illustrates how and when to use parallelization in mlr3.
+
+**Database Backends**
+
+The Section [Database Backends](#backends) describes how to work with database backends that `r mlr_pkg("mlr3")` supports.
```
Comment (pat-s) [inline bookdown/04-technical/04-technical.Rmd]:
```suggestion
The section [Database Backends](#backends) describes how to work with database backends that `r mlr_pkg("mlr3")` supports.
```

File: bookdown/04-technical/04-technical.Rmd [hunk 6]
```diff
@@ -2,12 +2,39 @@
...
+
+**Parameters**
+
+In the section [Parameters](#paradox) instructions are given on how to:
+
+* Define parameter sets for learners
+* Undertake parameter sampling
+* Apply parameter transformations
+
+For illustrative purposes, this sub-chapter uses the package `paradox`, the successor of the package `ParamHelpers`.
```
Comment (pat-s) [inline bookdown/04-technical/04-technical.Rmd]:
```suggestion
For illustrative purposes, this sub-chapter uses the `paradox` package, the successor of `ParamHelpers`.
```

File: bookdown/04-technical/04-technical.Rmd [hunk 7]
```diff
@@ -2,12 +2,39 @@
...
+
+**Logging and Verbosity**
 
-Section [Database Backends](#backends) describes how to work with database backends that `r mlr_pkg("mlr3")` supports.
-Database backends can be helpful for large data processing which does not fit in memory or is stored natively in a database (e.g. sqlite).
+The sub-chapter on [Logging and Verbosity](#logging) shows how to change the most important settings related to logging.
+For this sub-chapter we use the package _lgr_.
```
Comment (pat-s) [inline bookdown/04-technical/04-technical.Rmd]:
```suggestion
In _mlr3_ we use the _lgr_ package.
```

File: bookdown/04-technical/04-technical.Rmd (hunk 2 above)
Comment (danielsaggau) [reply bookdown/04-technical/04-technical.Rmd]: Good point.
Would computational power / computational capacity / computational energy fit better?

File: bookdown/04-technical/04-technical.Rmd (hunk 3 above)
Comment (danielsaggau) [reply bookdown/04-technical/04-technical.Rmd]: Yeah, I agree! It does sound overly negative. I changed it now.

File: bookdown/04-technical/04-technical.Rmd (hunk 2 above)
Comment (pat-s) [reply bookdown/04-technical/04-technical.Rmd]: I've never head this terms before in that context. Usually people use "computing power" or "computing resources". What do you think?

**Comment by danielsaggau** [comment]
I changed it to computing power. It sounds most suited to me.

**Pull request merged**